nemo/collections/nlp/modules/common/megatron/retrieval_transformer.py

```diff
@@ -138,28 +149,103 @@ def forward(
         encoder_output=None,
         layer_past=None,
         get_key_value=False,
+        set_inference_key_value_memory=False,
```

Comment:
Can you add comment describing `set_inference_key_value_memory` and `inference_max_sequence_len`

Reply:
added